src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -5,20 +5,28 @@
 using Avalonia.Diagnostics.Views;
 using Avalonia.Input;
 using Avalonia.Interactivity;
-
+#nullable enable 
 namespace Avalonia.Diagnostics
 {
-    public static class DevTools
+    internal static class DevTools
```

Comment:
This is a breaking change, meaning that we couldn't merge this in the 0.10 releases.